src/Shared/Helpers/SarifBuilder.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Microsoft.CodeAnalysis.Sarif;
+using NLog;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Text;
+
+namespace Microsoft.CST.OpenSource.Shared
+{
+    public class SarifBuilder
+    {
+        /// <summary>
+        /// Class logger
+        /// </summary>
+        protected static readonly NLog.Logger Logger = NLog.LogManager.GetCurrentClassLogger();
+        const string SARIF_V1_STRING = "sarifv1";
```

Comment:
These three values look like an enum to me.  You can use enum.tryparse to capture when users change capitalization etc below instead of ==, and you can store an enum value instead of a string.

Reply:
makes sense, will convert to enum